### PR TITLE
Stop 403 showing for other error codes

### PR DESCRIPTION
**Pull request by MichaelJacob21012**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/EM-5406

### Change description ###
Stop 403 showing for other error codes

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/em/orchestrator/config/security/SecurityConfiguration.java [hunk 1]
```diff
@@ -53,12 +54,13 @@ public WebSecurityCustomizer webSecurityCustomizer() {
 
     @Bean
     public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
-        http.csrf(csrf -> csrf.disable())
-                .formLogin(login -> login.disable())
-                .logout(logout -> logout.disable())
+        http.csrf(AbstractHttpConfigurer::disable)
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/em/orchestrator/config/security/SecurityConfiguration.java:57]: is this related to 403?

**Review by mdayican — APPROVED** [review]

**Review by yogesh-hullatti — APPROVED** [review]
